src/portals/hope/features/claims/claim-details/ClaimSubclaims/ClaimSubclaims.tsx

```diff
@@ -160,7 +170,13 @@ export const ClaimSubclaims: React.FC<{ claimId: string }> = ({ claimId }) => {
 
```

Comment:
I have a faint memory that this is the correct deps
```suggestion
 }, [subclaims?.length])
```